montara/des_tile.py

```diff
@@ -626,6 +626,11 @@ def setup(self, config, base, file_num, logger):
         if not isinstance(rs, list):
             first = galsim.config.ParseValue(
                 base['image'], 'random_seed', base, int)[0]
+
+            # launder through the RNG a few times to randomize
+            for _ in range(10):
+                first = galsim.BaseDeviate(first).raw()
```

Comment:
"a few times" is silly.  There's absolutely no reason to do it more than once.

Reply:
How does this fix the problem of the seeds?